### DISCUSSION TITLE
Change password or  reset password link of pimcore redirects to login page instead profile page

### DISCUSSION THREAD

**Discussion by shubhangi92** [pimcore/pimcore#11646] [General]
Hi,

We are using pimcore function for generating reset password link and send via email to user, but when copy the link and paste in browser its working but click on this is not working, redirecting to login page not user profile page.

Please help with this.
![image_2022_03_11T07_18_12_548Z](https://user-images.githubusercontent.com/80772901/157820754-f9550181-d37a-4db0-ad77-3df08d96da4c.png)

Thanks

**Comment by prateeksahus** [comment]
Hi @shubhangi92 I am also facing the same problem, it seems to be due to gmail's **data-saferedirecturl** attribute in the link. Did you find any workaround for this?

**Comment by fcaps** [comment]
Looks like a security/cookie issue of some sort. could confirm that the redirect happens because the PHPSESSID was getting filtered by the browser, thus the browser is loosing the logged-in-session and getting redirected to the login-page.
If i copy the link into a new tab this is not the case. Tested in FF and Chrome.

**Reply by HaraldDe** [reply]
Same issue

**Reply by mmodmmod** [reply]
> Looks like a security/cookie issue of some sort. could confirm that the redirect happens because the PHPSESSID was getting filtered by the browser, thus the browser is loosing the logged-in-session and getting redirected to the login-page. If i copy the link into a new tab this is not the case. Tested in FF and Chrome.

In our case, it turned out to be a Gmail issue. Apparently Gmail redirects to the login page. Without Thunderbird and Microsoft Teams we did not face these issues
